hugo/content/blog/2020-12-24-mumble-1-4-0-snapshot.md

```diff
@@ -0,0 +1,157 @@
+---
+title: Mumble 1.4.0 Snapshot Announcement
```

Comment:
We called them _Development Snapshot_ and _Snapshot_ before. I think the development snapshot is a very useful distinction that clarifies what it is.

We could also call it "first development snapshot" to indicate this is the start of them, and not a singular release.